samples/basic/servo_motor/sample.yaml

```diff
@@ -5,4 +5,4 @@ tests:
     tags: drivers pwm
     depends_on: pwm
     harness: motor
-    filter: dt_alias_exists("pwm-servo")
+    platform_whitelist: bbc_microbit
```

Comment:
Why do this as a whitelist instead of filter?

Reply:
Does the whitelist work even though the alias is defined in an overlay?